packages/accounts-controller/src/index.ts

```diff
@@ -8,10 +8,14 @@ export type {
   AccountsControllerGetSelectedAccountAction,
   AccountsControllerGetAccountByAddressAction,
   AccountsControllerGetAccountAction,
+  AccountsControllerGetNextAvailableAccountNameAction,
+  AccountsControllerGetSelectedMultichainAccountAction,
```

Comment:
Maybe we can move those into a separate PR (since we also miss `AccountsControllerListMultichainAccountsAction` too)

WDYT?